src/mpi/coll/src/coll_impl.c

```diff
@@ -956,7 +956,7 @@ int MPII_Coll_init(void)
         MPIR_Scatterv_inter_algo_choice = MPIR_SCATTERV_INTER_ALGO_AUTO;
 
     /* register non blocking collectives progress hook */
-    mpi_errno = MPID_Progress_register_hook(MPIDU_Sched_progress, &MPIR_Nbc_progress_hook_id);
+    mpi_errno = MPID_Progress_register_hook(MPIDU_Sched_list_progress, &MPIR_Nbc_progress_hook_id);
```

Comment:
This is not a `sched_list` function.  It's not specific to a list or to a schedule.  How about we use the following naming:

`MPIR|MPIDU_Sched_element_*`
`MPIR|MPIDU_Sched_list_*`
`MPIR|MPIDU_Sched_global_*`

Reply:
Just a note following our offline discussion. The list of all schedules is global and so `Sched_global` and `Sched_list` mean the same thing.

Using the following naming instead:
`MPIR|MPIDU_Sched_element_*`
`MPIR|MPIDU_Sched_list_*`